client/lib/domains/registration/availability-messages.js

```diff
@@ -366,6 +366,19 @@ function getAvailabilityNotice( domain, error, errorData ) {
 			);
 			break;
 
+		case domainAvailability.AVAILABLE_RESERVED:
+			message = translate(
+				'Sorry, {{strong}}%(domain)s{{/strong}} is reserved by the .%(tld)s registry and cannot be registered without permission.',
+				{
+					args: { domain, tld },
+					components: {
+						strong: <strong />,
+						a: <a rel="noopener noreferrer" href={ domainMapping( site, domain ) } />,
```

Comment:
I believe the `a` component can be removed 🙂

Reply:
🤦‍♂️ Thanks!